inference-engine/samples/speech_sample/main.cpp

```diff
@@ -665,8 +665,29 @@ int main(int argc, char *argv[]) {
         }
         auto t0 = Time::now();
         ExecutableNetwork executableNet;
-
         if (!FLAGS_m.empty()) {
+            if (!FLAGS_oname.empty()) {
+                std::string outputs_str = FLAGS_oname;
+                std::string outputLayer;
+                std::vector<std::string> outputs;
+                std::vector<size_t> ports;
+                size_t pos;
+                while ((pos = outputs_str.find(",")) != std::string::npos) {
+                    outputLayer = outputs_str.substr(0, pos);
+                    size_t pos_layer = outputLayer.rfind(":");
+                    outputs.push_back(outputs_str.substr(0, pos_layer));
+                    ports.push_back(std::stoi(outputLayer.substr(pos_layer + 1, outputLayer.length() - 1)));
```

Comment:
`std::stoi` can throw exception if value is not an integer or not in range. We should handle this case and print some meaningful error.